src/extension/features/toolkit-reports/pages/net-worth/components/legend/component.jsx

```diff
@@ -23,6 +23,14 @@ export const Legend = props => (
         <Currency value={props.assets} />
       </div>
     </div>
+    <div className="tk-debt-ratio">
```

Comment:
based on comment above, ya id just wrap this all with:

```jsx
import { isFeatureEnabled } from 'toolkit/extension/utils/feature';

...

{isFeatureEnabled(window.ynabToolKit.options.DebtRatio) && (
  <div>
    ...
  </div>
)}
```

Reply:
That is much more elegant! I have changed it to conditional rendering.